client/src/components/home/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import HomeLayout from '../home-layout';
+
+function Home() {
+  return (
+    <>
```

Comment:
Why angular brackets when you have only single child?

Reply:
Removed the brackets